docs/source/_sidebar.rst.inc

```diff
@@ -0,0 +1,18 @@
+
```

Comment:
Do we need this file in git?

Reply:
Removed from git tracking since it is autogenerated